paf-mvp-demo-express/src/views/publisher/index.hbs

```diff
@@ -749,8 +749,26 @@
                         <li><a href="#">User Agreement</a></li>
                         <li><a href="#">Privacy</a></li>
                         <li><a href="#">About us</a></li>
-                        <li><a href="#">Contact</a></li>
+                        <li><a href="#" id="preferences" class="d-flex align-items-center">
+                          <div class="mr-2">
+                            <img src="/assets/publisher/images/Union.svg" alt="">
+                          </div>
+                          <div>
+                            Marketing preferences
+                          </div>
+                        </a></li>
                     </ul>
+                  <script>
+                    document.querySelector('#preferences')
+                      .addEventListener('click', (e) => {
+                        e.preventDefault();
+                        if (PAF.getIdsAndPreferences()) {
+                          PAFUI.promptConsent();
+                        } else {
+                          CMP.cmpCheck({ proxyHostName: "{{proxyHostName}}", forcePrompt: true});
```

Comment:
Sorry to discuss it again, but I have the feeling we should ALWAYS do this:
even if there are some local cookies set, it feels safer to call `cmpCheck` with `forcePrompt: true`, because it will always refresh data from the operator, which is a good practice to make sure we show up to date values to the end user.
WDYT?

Reply:
I think it requires a lot of changes. First of all, cmpCheck should ignore fresh cookies and show the popup. But the main issue is with the no-3PC flow. Trying refresh settings we will exit from cmpCheck.
It will be overcomplicated.

Reply:
I agree the main concern is about no-3PC and redirects.
Let's merge it as-is and iterate in the future, if needed.
Thanks for your answer.